docs/network/default_cni_plugin.md

```diff
@@ -149,3 +150,44 @@ The specified interface will be added in OVS bridge `br-ex` which acts as gatewa
 microshift-ovs-init.service is able to setup one additional host interface for cluster ingress/egress traffic.
 This is done by specifying the `externalGatewayInterface` in the CNI config file `/etc/microshift/ovn.yaml`.
 The external gateway interface will be added in a second OVS bridge `br-ex1`. Cluster pod traffic destinated to additional host subnet will be routed through `br-ex1`.
+
+### NodePort host addresses
+
+ovn-kubernetes doesn't restrict the host interfaces where nodePort service can be accessed from outside MicroShift node. The following `nft` instructions block nodePort service on a specific host interface.
+
+Find the ovn-kubernetes nodeport ruleset `handle` in nftable `ip nat` table `OVN-KUBE-NODEPORT` chain:
+```text
+(host)$ nft -a list table ip nat | grep "chain OVN-KUBE-NODEPORT" -A3
+	chain OVN-KUBE-NODEPORT { # handle 22
+		meta l4proto tcp fib daddr type local tcp dport 30700 counter packets 28 bytes 1680 dnat to 10.43.138.214:7000 # handle 56
+	}
+```
+
+> Record the `handle` number `56`, nodeport number `30700`
+
+<br>
+
+Insert a new rule before the ovn-kubernetes nodeport rule to `drop` the packet with matching destination port and ip:
+```text
+(host)$ nft -a insert rule ip nat OVN-KUBE-NODEPORT position 56 tcp dport 30700 ip daddr 192.168.150.33 drop
+```
+> Replace the handle number (after position) and nodeport number (after dport) with recorded values in the above step
+> Replace `192.168.150.33` with the IP address from the host interface where you'd like to block the nodeport service
```

Comment:
````suggestion
```text
(host)$ HANDLE=56
(host)$ NODEPORT=30700
(host)$ INTERFACE_IP=192.168.150.33
(host)$ nft -a insert rule ip nat OVN-KUBE-NODEPORT position $HANDLE tcp dport $NODEPORT ip daddr $INTERFACE_IP drop
```
> Replace value of HANDLE and NODEPORT variables with recorded values in the above step
> Replace value of INTERFACE_IP with the IP address from the host interface where you'd like to block the NodePort service
````

WDYT?

Reply:
sure, that looks clean and better.